spec/cypress/integration/nominations/Nominations.feature

```diff
@@ -30,4 +30,4 @@ Feature: Resend nominations flow
     Then the page should be accessible
 
     When I am on "resend nominations limit reached" page
-    Then the page should be accessible
+    Then the page should be accessible
```

Comment:
If there's now more than one nominations feature spec, should this be renamed to something a bit more specific than Nominations.feature?

Reply:
Changed the name. Done